ws/websocket.go

```diff
@@ -33,6 +34,10 @@ const (
 	defaultHandshakeTimeout = 30 * time.Second
 	// Default sub-protocol to send to peer upon connection.
 	defaultSubProtocol = "ocpp1.6"
+	// The base delay to be used for automatic reconnection. Will increase exponentially up to maxReconnectionDelay.
+	defaultAutoReconnectDelay = 5 * time.Second
+	// Default maximum reconnection delay for websockets
+	defaultMaxReconnectionDelay = 2 * time.Minute
```

Comment:
same here. 2 minutes seems rather conservative?

Reply:
the value represents the amount of time that passes before attempting a reconnection. Combined with `defaultAutoReconnectDelay`, basically the flow would be:
- disconnected
- attempt to re-connect after 5 seconds
- attempt to re-connect after 10 seconds
- attempt to re-connect after 20 seconds
- attempt to re-connect after 40 seconds
- attempt to re-connect after 80 seconds
- attempt to re-connect after 120 seconds (`defaultMaxReconnectionDelay`, won't go higher than this)
- attempt to re-connect after 120 seconds
- attempt to re-connect after 120 seconds
- re-connect successful!

I chose the values considering real world scenarios where charge points use sim-cards and don't always have good coverage. If you believe these are too high, I can set them lower.

Also, I'll add the possibility to configure them through the `ClientTimeoutConfig` struct, so should be fine.

Reply:
Renamed the variables to be more intuitive.
Also now uses the user-configured ones, instead of the default ones.